TrackerGeom/inc/Tracker.hh

```diff
@@ -48,7 +48,7 @@ namespace mu2e {
     using StrawCollection = std::array<Straw,StrawId::_nustraws>;
     using StrawIndexMap = std::array<uint16_t,StrawId::_maxval>; 
     // default constructor results in non-functional object, but is required by proditions service
-    Tracker() : _name("AlignedTracker") {}
+    Tracker() : _name("Tracker") {}
```

Comment:
Does it matter that the GeometryService Tracker object has the same '_name'?  Does this string have to match between the TrackerCache and Tracker object?  If so, is there a way to synchronize them, maybe through
 a constexpr?

Reply:
The names in the proditions object and the proditions cache have to be the same.  It might be possible to drive both off one constexpr.  I can try it.

Reply:
Can we drive this off of the boost type_id rather than having people hand code strings?